spec/rubocop/cop/rspec/void_expect_spec.rb

```diff
@@ -44,4 +44,47 @@
       end
     RUBY
   end
+
+  it 'ignores unrelated method named expect in an example block' do
+    expect_no_offenses(<<~RUBY)
+      it 'something' do
+        MyObject.expect
+      end
+    RUBY
+  end
+
+  it 'flags bare `expect` in example block' do
+    expect_offense(<<~RUBY)
+      it 'something' do
+        expect
```

Comment:
This behavior is also debatable. I'm somewhat in favor of leaving it. 

I think this would only be possible if someone is shadowing (overwrote in some context) the `expect` method. In that case, ideally there would be an even more specific error message, but I also don't think it's a case that deserves its own cop. 

So the fact that there is a Linter violation here is warranted I think, even if the message is not directly addressing how strange this is.

Reply:
Completely agree, a “bare” is [invalid](https://github.com/rspec/rspec-expectations/blob/838952dc1416c943e7933684c47249a77481e7f5/lib/rspec/expectations/expectation_target.rb#L39) in RSpec.

Do you think we can just remove those examples with “bare” expect calls?

Reply:
I guess the only reason to keep this and essentially document this behavior is if we thought it were worth changing. 

Since we both agree it's not worth changing, I will go ahead and delete it. 

I still feel like we "know less" about how the Cop works with out this -- in the sense that I learned something more about how it worked by writing this, but that's perhaps not a strong enough reason to commit it since it's covered by RSpec behavior (or the user did something very strange).